frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard.tsx

```diff
@@ -71,7 +77,7 @@ export const CreateInternalCluster: React.FC<CreateInternalClusterProps> = ({ ma
       name: 'Configure',
       id: CreateStepsSC.CONFIGURE,
       component: <Configure state={state} dispatch={dispatch} mode={mode} />,
-      enableNext: state.encryption.hasHandled && hasConfiguredNetwork,
+      enableNext: state.encryption.hasHandled && hasConfiguredNetwork && state.kms.hasHandled,
```

Comment:
Please remove any restriction from individual steps, until we push validation based wizard.
```suggestion
```

Reply:
It was decided to have `Next` button blocked wrt validations in the wizard.
@cloudbehl @yuvalgalanti

Reply:
yes that's what was planned.

Reply:
We have navs still clickable, I thnk we should block them as well.
(Possibly in another PR)

Reply:
+1 @afreen23